<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>PermsDialogUI</class>
 <widget class="QDialog" name="PermsDialogUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Permissions</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="RecordTreeView" name="mTreeView" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="mNewMethodButton" >
       <property name="text" >
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mRemoveMethodButton" >
       <property name="text" >
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>121</width>
         <height>14</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOKButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>RecordTreeView</class>
   <extends>QTreeView</extends>
   <header>recordtreeview.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mOKButton</sender>
   <signal>clicked()</signal>
   <receiver>PermsDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>398</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel" >
     <x>345</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCancelButton</sender>
   <signal>clicked()</signal>
   <receiver>PermsDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>516</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel" >
     <x>516</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
